src/codec_aom.c

```diff
@@ -783,6 +783,9 @@ static avifResult aomCodecEncodeImage(avifCodec * codec,
         }
         minQuantizer = AVIF_CLAMP(minQuantizer, 0, 63);
         maxQuantizer = AVIF_CLAMP(maxQuantizer, 0, 63);
+        // Make sure that maxQuantizer >= quantizer >= minQuantizer.
```

Comment:
Add the following comment:

> If quantizer conflicts with minQuantizer and maxQuantizer, quantizer has precedence.

Note: The conflict could also be resolved by giving minQuantizer and maxQuantizer precedence:

```
        quantizer = AVIF_CLAMP(quantizer, minQuantizer, quantizer);
```

Since minQuantizer and maxQuantizer are deprecated, it doesn't make sense to give them precedence.